replacing broken link in README

RudyOnRails opened the pull request:
Not sure where the #web-server id went, but I thought I'd replace with another link to a Passenger tut.

@projecthydra/sufia-code-reviewers

RudyOnRails committed:
replacing broken link

Not sure where the #web-server id went, but I thought I'd replace with another link to a Passenger tut.

mjgiarlo commented:
@RudyOnRails Good eye!

Link moved out of README and into the Sufia Mgmt Guide, here: https://github.com/samvera/sufia/wiki/Sufia-Management-Guide#web-server

RudyOnRails committed:
Link moved out of README and into the Sufia Mgmt Guide

mjgiarlo commented 4 hours ago
@RudyOnRails Good eye!

Link moved out of README and into the Sufia Mgmt Guide, here: https://github.com/samvera/sufia/wiki/Sufia-Management-Guide#web-server

RudyOnRails commented:
Not sure why this is red when it's just the readme 🤔 . Maybe this is related to the failing master build that @jrochkind is working on.
